src/pages/AllDao.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<a-row>
+		<a-col :span="12" :offset="6">
+			<div v-if="error">
+				<a-result status="error" title="Submission Failed" :sub-title="error">
+				</a-result>
+			</div>
+			<div v-else>
+				<a-form
+					:label-col="{ span: 12 }"
+					:wrapper-col="{ span: 12 }"
+					:model="formState"
+					name="All Dao"
+					autocomplete="off"
+					@finish="onFinish"
+					@finishFailed="onFinishFailed"
+					@validate-messages="validateMessages"
+				>
+					<a-form-item
+						label="Select DAO"
+						name="dao_id"
+						:rules="[{ required: true, type: 'number' }]"
+					>
+						<a-input-number v-model:value="formState.dao_id" />
+					</a-form-item>
+
+					<a-form-item :wrapper-col="{ offset: 10, span: 20 }">
+						<a-button type="primary" html-type="submit">Select</a-button>
+					</a-form-item>
+				</a-form>
+			</div>
+		</a-col>
+	</a-row>
+</template>
+
+<script lang="ts">
+import { validateMessages } from "@/constants/constant";
+import { searchForAccount, searchForApplication } from "@/indexer";
+import WalletStore from "@/store/WalletStore";
+import { defineComponent, reactive } from "vue";
+import DaoStore from "../store/DaoID";
+
+export default defineComponent({
+	name: "AllDao",
+	data() {
+		return {
+			error: "",
+		};
+	},
+	methods: {
+		onFinish(values: any) {
+			searchForApplication(+values.dao_id)
+				.then((response) => {
+					console.log("Global App Info:", response);
+					if (response) {
+						this.formState.global_app_state = response;
+						this.formState.name = this.formState.global_app_state.get(
+							"dao_name"
```

Comment:
There are many such constant strings available. We should move them all to a constant file. Same local_app_state or wherever applicable.